packages/create-fuels/src/cli.ts

```diff
@@ -94,9 +94,23 @@ export const runScaffoldCli = async (
   shouldInstallDeps = true,
   explicitProgramsToInclude?: ProgramsToInclude
 ) => {
-  new Command(packageJson.name).version(packageJson.version);
-
-  const projectPath = explicitProjectPath || (await promptForProjectPath());
+  const program = new Command(packageJson.name)
+    .version(packageJson.version)
+    .arguments('[projectDirectory]')
+    .option('-C, --contract', 'Include contract program')
+    .option('-P, --predicate', 'Include predicate program')
+    .option('-S, --script', 'Include script program')
+    .option('--pnpm', 'Use pnpm as the package manager')
+    .option('--npm', 'Use npm as the package manager')
+    .option('--cs', 'Include contract and script programs')
+    .option('--cp', 'Include contract and predicate programs')
+    .option('--ps', 'Include predicate and script programs')
+    .option('--cps', 'Include all programs')
```

Comment:
This can be simplified to:

```suggestion
    .option('-c, --contract', 'Include only contract program')
    .option('-p, --predicate', 'Include only predicate program')
    .option('-s, --script', 'Include only script program')
    .option('--pnpm', 'Use pnpm as the package manager')
    .option('--npm', 'Use npm as the package manager')
```

The default is to generate all:

```shell
pnpm create fuels my-project # all
```

Users can opt-in to include only a subset:

```shell
pnpm create fuels my-project -c # contract only
pnpm create fuels my-project -p # predicate only
pnpm create fuels my-project -s # script only
```

Or any combo:

```shell
pnpm create fuels my-project -cp # contract + predicate
pnpm create fuels my-project -cs # contract + script
pnpm create fuels my-project -ps # predicate + script
pnpm create fuels my-project -cps # all, same as not informing anything
```

Reply:
Absolutely! This is valid for boolean flags. 🎉